src/map/entities/mobentity.cpp

```diff
@@ -986,15 +986,15 @@ void CMobEntity::DropItems(CCharEntity* PChar)
 
     bool validZone = ((Pzone > 0 && Pzone < 39) || (Pzone > 42 && Pzone < 134) || (Pzone > 135 && Pzone < 185) || (Pzone > 188 && Pzone < 255));
 
-    if (validZone && charutils::CheckMob(m_HiPCLvl, GetMLevel()) > EMobDifficulty::TooWeak)
+    if (!getMobMod(MOBMOD_NO_DROPS) && validZone && charutils::CheckMob(m_HiPCLvl, GetMLevel()) > EMobDifficulty::TooWeak)
```

Comment:
👍

Reply:
just want to point out that mob drops and aern crashes normally wouldn't go in the same pr, and I'd have rubber stamped this part in and merged already if it were standalone - its now held up by reviewing aern fixes